HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -1523,9 +1610,55 @@ def self.apply_hvac(runner, hpxml, weather, convert_shared_systems)
         HVAC.set_mshp_downselected_speed_indices(heat_pump)
 
       elsif [HPXML::HVACTypeHeatPumpGroundToAir].include? heat_pump.heat_pump_type
+        if heat_pump.geothermal_loop.nil?
+          hpxml.geothermal_loops.add(id: "GeothermalLoop#{hpxml.geothermal_loops.size + 1}",
+                                     loop_configuration: HPXML::GeothermalLoopLoopConfigurationVertical)
+          heat_pump.geothermal_loop_idref = hpxml.geothermal_loops[-1].id
+        end
+
+        if heat_pump.geothermal_loop.pipe_size.nil?
+          heat_pump.geothermal_loop.pipe_size = 0.75 # in
+          heat_pump.geothermal_loop.pipe_size_isdefaulted = true
+        end
+
         HVAC.set_gshp_assumptions(heat_pump, weather)
         HVAC.set_curves_gshp(heat_pump)
 
+        if heat_pump.geothermal_loop.bore_spacing.nil?
+          heat_pump.geothermal_loop.bore_spacing = 16.4 # ft, distance between bores
+          heat_pump.geothermal_loop.bore_spacing_isdefaulted = true
+        end
+
+        if heat_pump.geothermal_loop.bore_diameter.nil?
+          heat_pump.geothermal_loop.bore_diameter = 5.0 # in
+          heat_pump.geothermal_loop.bore_diameter_isdefaulted = true
+        end
+
+        if heat_pump.geothermal_loop.grout_type.nil? && heat_pump.geothermal_loop.grout_conductivity.nil?
+          heat_pump.geothermal_loop.grout_type = HPXML::GeothermalLoopGroutTypeStandard
+          heat_pump.geothermal_loop.grout_type_isdefaulted = true
+        end
+
+        if heat_pump.geothermal_loop.grout_conductivity.nil?
+          if heat_pump.geothermal_loop.grout_type == HPXML::GeothermalLoopGroutTypeStandard
+            heat_pump.geothermal_loop.grout_conductivity = 0.4 # Btu/h-ft-R
+            heat_pump.geothermal_loop.grout_conductivity_isdefaulted = true
+          elsif heat_pump.geothermal_loop.grout_type == HPXML::GeothermalLoopGroutTypeThermallyEnhanced
+            heat_pump.geothermal_loop.grout_conductivity = 0.8 # Btu/h-ft-R
+            heat_pump.geothermal_loop.grout_conductivity_isdefaulted = true
+          end
+        end
+
+        if heat_pump.geothermal_loop.pipe_cond.nil?
+          heat_pump.geothermal_loop.pipe_cond = 0.23 # Btu/h-ft-R; Pipe thermal conductivity, default to high density polyethylene
+          heat_pump.geothermal_loop.pipe_cond_isdefaulted = true
+        end
```

Comment:
This should be expanded to be more like how `group_type` and `grout_conductivity` are handled. E.g., if neither `pipe_type` or `pipe_conductivity` are provided, default to standard. Then standard maps to conductivity=0.23 and we should add a value for thermally enhanced.